core/src/main/scala/uk/gov/nationalarchives/tdr/api/core/graphql/service/FileStatusService.scala

```diff
@@ -0,0 +1,23 @@
+package uk.gov.nationalarchives.tdr.api.core.graphql.service
+
+import uk.gov.nationalarchives.tdr.api.core.FileStatus
+import uk.gov.nationalarchives.tdr.api.core.db.dao.FileStatusDao
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class FileStatusService(fileStatusDao: FileStatusDao)(implicit val executionContext: ExecutionContext) {
+  def updateChecksum(id: Int, checksum: String): Future[Boolean] = {
+    fileStatusDao.updateChecksum(id, checksum).map(a => {
+      println(a)
```

Comment:
Probably want to delete this logging.

Reply:
I've taken all the print statements out of this file.